test/support.js

```diff
@@ -107,34 +107,19 @@ givenWorkspaceFromTemplate = function(template, options, cb) {
   });
 };
 
-givenWorkspaceWithCustomDependencies = function(templateName, deps, cb) {
+givenLB2Workspace = function(cb) {
   resetWorkspace(function(err) {
     if (err) return cb(err);
-    givenWorkspaceFromTemplate('empty-server', deps, function(err) {
-      if (err) return cb(err);
-      ConfigFile.loadFromPath('/package.json', function(err, pkgFile) {
-        if (err) return cb(err);
-        for (dep in deps) {
-          pkgFile.data.dependencies[dep] = deps[dep];
-        }
-
-        pkgFile.save(cb());
-      });
-    });
+    givenWorkspaceFromTemplate('empty-server', cb);
```

Comment:
Please be explicit and set `options.loopbackVersion`. We will change the default from 2.x to 3.x in the near future, this helper should continue working even after that change.